projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -206,6 +224,25 @@ export default function PublicizeForm( {
 							) }
 						</ul>
 					</PanelRow>
+					{ connections.some( ( { service_name } ) => isConnectionNeedMedia( service_name ) ) &&
+						! postHasValidImage && (
+							<Notice type={ 'warning' }>
+								{ createInterpolateElement(
+									__(
+										'You need a valid image in your post to share to Instagram.<br/><fixLink>Learn more</fixLink>',
+										'jetpack'
+									),
+									{
+										br: <br />,
+										fixLink: (
+											<ExternalLink
+												href={ getRedirectUrl( 'jetpack-social-media-support-information' ) }
+											/>
+										),
+									}
+								) }
```

Comment:
To simplify things here a bit, we can use two separate strings here:
1. `__( 'You need a valid image in your post to share to Instagram.', 'jetpack' )`
2. `__( 'Learn more', 'jetpack' )`

It has a few benefits:
- We will not need to use `createInterpolateElement`
- `<br />` may not be necessary.
- We can style the items better without `<br />`
- Translation of `__( 'Learn more', 'jetpack' )` is already available and thus can be re-used.

```suggestion
								{ __( 'You need a valid image in your post to share to Instagram.', 'jetpack' ) }
								<br />
								<ExternalLink href={ getRedirectUrl( 'jetpack-social-media-support-information' ) }>
									{ __( 'Learn more', 'jetpack' ) }
								</ExternalLink>
```

Reply:
Makes sense. Thanks!